pkg/kubelet/dockershim/remote/docker_service.go

```diff
@@ -49,6 +49,10 @@ func (d *dockerService) Version(ctx context.Context, r *runtimeApi.VersionReques
 	return d.runtimeService.Version(r.GetVersion())
 }
 
+func (d *dockerService) Status(ctx context.Context, r *runtimeApi.StatusRequest) (*runtimeApi.StatusResponse, error) {
+	return nil, fmt.Errorf("not implemented")
```

Comment:
Yes, we probably need to change [`syncNetworkStatus`](https://github.com/kubernetes/kubernetes/blob/2941aea88e80e94f2698241a4ac9925145664470/pkg/kubelet/kubelet_network.go#L193) to pipe this information.